### DISCUSSION TITLE
Can Crosssplane be used for on-Premis activities

### DISCUSSION THREAD

**Discussion by eliassal** [crossplane/crossplane#4915] [General]
I have a 3 node Kubernetes cluster up and running on prem and we never use cloud for the time being, all ops are on prem. Can Crossplane be usefull and help us generate infrastructure on-prem and not on the cloud? Thanks

**Comment by bobh66** [comment]
You can use provider-terraform, provider-kubernetes and provider-helm to work on local infrastructure without using a cloud provider.

**Comment by eliassal** [comment]
So many thanks for your response, any tutorial or step-by-step instructions you point me to to acheive this?
Thanks again
